includes/class-wc-payments-account.php

```diff
@@ -1794,7 +1795,7 @@ public function get_account_country() {
 	 */
 	public function get_account_default_currency() {
 		$account = $this->get_cached_account_data();
-		return $account['store_currencies']['default'] ?? 'usd';
+		return $account['store_currencies']['default'] ?? strtolower( Currency_Code::USD );
```

Comment:
Found to a bug, thanks to @achyuthajoy

Reply:
Fixed https://github.com/Automattic/woocommerce-payments/pull/7997/commits/d633c6640977d38c82c8bfdf9bb1e9fd47e8c084